getting_started/escoria_nodes.rst

```diff
@@ -0,0 +1,135 @@
+
+An ESCItem is accessible in Escoria scripts by the name provided in the
+`Global ID` field. This name should be unique to your whole game.
+
+You will typically assign a few child nodes to each ESCItem.
+
+* Sprite / Animated Sprite : This is an image of the object that will be
+  displayed on the screen for the player.
+* CollisionPolygon2D : This is a shape that tells Escoria when the mouse is
+  over the object. Set this up to cover the Sprite.
+* ESCLocation : If you make an ESCLocation the child of an ESCItem, when you
+  interact with the item, the character will automatically walk to this
+  position to perform the interaction. eg. place an ESCLocation underneath a
+  "window" ESCItem and the player will walk to the window when you "look" at
+  it.
+
+.. _`ESCItem`: https://docs.escoria-framework.org/en/devel/api/ESCItem.html#
+
+ESCBackground
+~~~~~~~~~~~~~
+
+This is the background image for your room. ESCItems will be drawn over the
+top of this. Assign the background image for the room to the ESCBackground's
+"texture"
+
+.. _`ESCBackground`: https://docs.escoria-framework.org/en/devel/api/ESCBackground.html
+
+ESCTerrain
+~~~~~~~~~~
+
+An ESCTerrain tells Escoria where in the scene the character is allowed to
+walk. When clicking within the game window, the character will automatically
+walk to the clicked spot (or as close to it as it can) while staying
+within the terrain.
+
+The game allows for multiple terrains to be configured, but only one can be
+active at a time. A good use for multiple terrains would be a hallway with a
+closed door in the middle. The initial terrain would cover the floor on the
+side of the hallway where the player currently stood up to the doorway. This
+would allow them to walk up to the door, but not pass through or beyond it.
+
+A second terrain would include the area covered by the inital terrain, as well
+as the floor through the doorway and the far side of the hallway.
+
+In your script attached to the door ESCItem, when the player
+opened the door, you'd disable the first terrain and enable the second,
+allowing the game character to walk through the entire hallway.
+
+Terrains are a `NavigationPolygonInstance` node and are referenced in code by
+their name rather than a `Global ID`.
+
+.. _`ESCTerrain`: https://docs.escoria-framework.org/en/devel/api/ESCTerrain.html
+
+ESCLocation
+~~~~~~~~~~~
+
+An ESCLocation is a location within your game that the character will walk to
+under certain conditions.
+
+* You may use a scripted `walk` command to force the character to walk to a
+  certain location. This is handy for cutscenes as part of telling your game's
+  story.
+* If the ESCLocation is a child of an ESCItem, the character will automatically
+  walk to the location when you interact with the item.
+* If you call an ESCLocation "player_start", Escoria will automatically place
+  the player there when it loads a room (unless otherwise instructed by the
+  script attached to the ESCRoom).
```

Comment:
```suggestion
`ESCLocation`
~~~~~~~~~~~

An `ESCLocation` is a location within your game that the character will walk to
under certain conditions.

* You may use a scripted `walk` command to force the character to walk to a
  certain location. This is handy for cutscenes as part of telling your game's
  story.
* If the `ESCLocation` is a child of an `ESCItem`, the character will automatically
  walk to the location when you interact with the item.
* If you call an `ESCLocation` "player_start", Escoria will automatically place
  the player there when it loads a room (unless otherwise instructed by the
  script attached to the `ESCRoom`).
```